progress-bar-snippet.md

omerjava opened the pull request:
<!--
  make this PR easy to find:

  - assign and add reviewers
  - add any helpful labels
  - connect it to a milestone (if necessary)
  - link it with an issue (if necessary)
-->

<!-- describe your PR -->

## Checklists

### General Checks

- [x] the branch is up to date with `main`/`master`
- [x] the code works when pulled and run locally
- [x] All CI checks pass
- [x] all conflicts are resolved (if any)
- [x] PR has a descriptive title
- [x] PR has appropriate labels and milestones for easy identification
- [x] PR it is assigned to the owner
- [x] reviewers are assigned
- [x] the PR contributes only one focused change
- [x] It is in the appropriate column in the project board (if necessary)
- [x] has short and clear description
- [x] is linked to an issue (if it is related)
- [x] feedback is addressed (if any and if it is appropriate feedback.)

### Markdown

<!-- markdown-specific checks -->

- [x] the markdown source is formatted
- [x] spelling and grammar is correct in all text
- [x] The markdown looks correct when you preview the file
- [x] all links and images work
- [x] any embedded code snippets are formatted

arnochauveau commented on snippets/javascript/progress-bar-snippet.md:
```diff
@@ -0,0 +1,32 @@
+# A progress bar snippet with JavasScript and HTML
+
+## HTML code
+
```
Hey @omerjava small tip: in MD you can add syntax highlighting to code blocks: 
``` 
           ``` html
               <!-- your html here -->
            ```
            ``` js
             // your JS here
            ```
 ```
 
 which will look like this: 
 ```html
<!-- your html here -->
```
```js 
// your js here
```

or for your example: 

```html
<!DOCTYPE html>
    <html>
    <head>
        <meta charset="utf-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <title>Progress Bar</title>
        <meta name="viewport" content="width=device-width, initial-scale=1" />
        <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
        <script src="script.js"></script>
    </head>
    <body>
        <label for="file">Downloading progress:</label>
        <progress value="0" max="100" id="progressBar"></progress>
    </body>
    </html>
```

```js
var timeLeft = 100;
    var downloadTimer = setInterval(function() {
        if (timeLeft <= 0) {
        clearInterval(downloadTimer);
        }
        document.getElementById("progressBar").value = 100 - timeLeft;
        timeLeft -= 1;
    }, 100);
```

arnochauveau reviewed (changes requested):
Nice snippet, @omerjava. 
I've added some tips for formatting your MD file. 
Also try to pay attention to indenting to make your code more readable (for example the body of the if-statement) and your usage of var instead of let and const (bad practice to use var).

omerjava commented:
> Nice snippet, @omerjava. I've added some tips for formatting your MD file. Also try to pay attention to indenting to make your code more readable (for example the body of the if-statement) and your usage of var instead of let and const (bad practice to use var).

@arnochauveau Thanks for the tips. I updated it!

arnochauveau commented:
> > Nice snippet, @omerjava. I've added some tips for formatting your MD file. Also try to pay attention to indenting to make your code more readable (for example the body of the if-statement) and your usage of var instead of let and const (bad practice to use var).
> 
> @arnochauveau Thanks for the tips. I updated it!

Hey @omerjava the 

html here and js here snippets were as an example, you don't need to add those to use the syntax highlighting 🙂

I've removed those for you, take a look at my commit to see what I mean.

omerjava commented:
> > > Nice snippet, @omerjava. I've added some tips for formatting your MD file. Also try to pay attention to indenting to make your code more readable (for example the body of the if-statement) and your usage of var instead of let and const (bad practice to use var).
> > 
> > 
> > @arnochauveau Thanks for the tips. I updated it!
> 
> Hey @omerjava the
> 
> html here and js here snippets were as an example, you don't need to add those to use the syntax highlighting 🙂
> 
> I've removed those for you, take a look at [my commit](https://github.com/lab-antwerp-1/home/pull/194/commits/c692412042fc9dd2d72d5eb9dcece559eb7cf128) to see what I mean.

Ok 🙂 Thank you!
